app/src/main/java/org/jellyfin/androidtv/details/actions/ResumeAction.kt

```diff
@@ -0,0 +1,38 @@
+package org.jellyfin.androidtv.details.actions
+
+import android.content.Context
+import android.content.Intent
+import android.drm.DrmStore
+import android.util.Log
+import androidx.core.content.ContextCompat.startActivity
+import kotlinx.android.synthetic.main.text_under_button.view.*
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.TvApp
+import org.jellyfin.androidtv.playback.MediaManager
+import org.jellyfin.androidtv.util.TimeUtils
+import org.jellyfin.androidtv.util.apiclient.PlaybackHelper
+import org.jellyfin.apiclient.interaction.Response
+import org.jellyfin.apiclient.model.dto.BaseItemDto
+import org.jellyfin.apiclient.model.dto.BaseItemType
+
+private const val LOG_TAG = "ResumeAction"
+
+class ResumeAction(context: Context, val playbackPositionTicks: Long, val itemID: String) : PlaybackAction(ActionID.RESUME.id, context) {
+	val actualPlaybackPositionInMillis: Long
+
+	init {
+		this.actualPlaybackPositionInMillis = playbackPositionTicks / 10000 - TvApp.getApplication().resumePreroll
+		this.label1 = context.getString(R.string.lbl_resume_from, TimeUtils.formatMillis(actualPlaybackPositionInMillis))
+	}
+
+	override fun onClick() {
```

Comment:
```kotlin
override fun onClick() = GlobalScope.launch(Dispatchers.Main) {
	Log.i(LOG_TAG, "Resume Clicked!")
	playItemWithID(itemID, actualPlaybackPositionInMillis, false)
}
```

Reply:
Hmm, like this it complains that the Return Type is a Job, so it seems like we would also need to adjust the base class. Shall we do this?

Reply:
Oh in that case we can do 2 things:

End the expression with `Unit` (the void type of Kotlin) or don't change it.

Reply:
Okay, thanks. Will leave it.